src/common/ui-wrapper.js

```diff
@@ -58,6 +66,19 @@ class UIWrapper {
     this.setConfig({show: show}, 'loading');
   }
 
+  _handleExternalCSS(config: UIOptionsObject = {}): void {
+    if (config.css) {
+      Utils.Dom.loadStyleSheetAsync(config.css).then(
+        () => {
+          logger.debug(`external css was loaded successfully`);
+        },
+        () => {
+          logger.error(`external css failed to load`);
```

Comment:
Is it needed to maybe add some small retry logic if the loading has been failed?

Reply:
As we discussed this morning this is just a link with a source. If want to add retry mechanism we need to make something generic for all loadings